src/community/advanced-data-card.tsx

```diff
@@ -7,8 +7,7 @@ import Stack from '../stack';
 import * as styles from './advanced-data-card.css';
 import * as mediaStyles from '../image.css';
 import Divider from '../divider';
-import {Text2, Text} from '../text';
-import * as textProps from '../text-props';
```

Comment:
the problem was that some components were using `textProps` directly from `text-props.tsx` instead of reading them them from `useTheme`, the problem with this is they were using the default size values for text presets, so the size overrides made by the different skins were not being taking into account